docs/pbc/all/tax-management/202400.0/base-shop/vertex/install-vertex.md

```diff
@@ -491,16 +505,13 @@ class TaxAppDependencyProvider extends SprykerTaxAppDependencyProvider
      */
     protected function getCalculableObjectExpanderPluginCollection(): array
     {
-        return [
-            new CalculableObjectCustomerWithTaxCodeExpanderPlugin(), // to extend quote with customer class code
-            new CalculableObjectCustomerWithVertexExemptionCertificateExpanderPlugin(), // to extend quote with customer exemption certificate
-            new CalculableObjectExpensesWithTaxCodeExpanderPlugin(), // to extend quote expenses with product class codes
-            new CalculableObjectItemProductOptionWithTaxCodeExpanderPlugin(), // to extend quote item product options with product class codes
-            new CalculableObjectItemWithProductClassCodeExpandePlugin(), // to extend quote items with product class codes
-            new CalculableObjectItemWithFlexibleFieldsExpanderPlugin(), // to extend quote items with flexible fields
```

Comment:
why these are removed and others for the marketplace are left?

Please also check the paragraph, that plugins were referring to plugins from `spryker/tax-app-vertex`
```
2. Implement Vertex-specific metadata extender plugins

There are several types of expander plugins you have to introduce. As a starting point, you can take examples provided by Spryker in the [tax-app-vertex] module (https://github.com/spryker/tax-app-vertex). The plugins inside are for development purposes. The data in the TaxMetaData fields has to be collected from the project database or other sources such as external ERP.
```

Reply:
those are not marketplace-specific but are instead covering the plugins that are supposed to provide additional data that were described above. 
IMO those plugins should not be included in the plugin stack as they should only serve as an example. I have added a comment stating that all custom-implemented plugins should go into those plugin stacks.

Reply:
OK, let's also additional comment about checking the examples of expanders in `spryker/tax-app-vertex`.

Reply:
There is a mention of `spryker/tax-app-vertex` at the beginning of previous section which describes implementing custom project-level expanders, gives examples and refers to tax-app-vertex:

> There are several types of expander plugins you have to introduce. As a starting point, you can take examples provided by Spryker in the [tax-app-vertex](https://github.com/spryker/tax-app-vertex) module. The plugins inside are for development purposes. The data in the TaxMetaData fields has to be collected from the project database or other sources such as external ERP.

I can change the inline comment at the beginning of this plugin stack return value to mention tax-app-vertex as well.